package.json

```diff
@@ -3,6 +3,9 @@
 	"version": "0.3.0",
 	"description": "A command line interface for Lisk",
 	"main": "./dist/index.js",
+	"engines": {
+		"node": "=6.12.3"
```

Comment:
Can you update the `.nvmrc` file and add an engines entry for npm while we're here?